lib/pos_constants.go

```diff
@@ -0,0 +1,54 @@
+package lib
+
+import (
+	"github.com/deso-protocol/core/bls"
+	"github.com/deso-protocol/core/collections/bitset"
+	"github.com/deso-protocol/core/consensus"
+	"github.com/holiman/uint256"
+	"github.com/pkg/errors"
+)
+
+// The Proof of Work -> Proof of Stake cutover requires a synthetic QC to protect against a timeout
+// during the exact point of the cutover. This synthetic QC is built and signed locally by every node
+// using a known and consistent private key.
+const proofOfStakeCutoverValidatorBLSPrivateKeyHex = "0x0570b78ce822f902b203ee075a7e2147d6b9a420a9409c038154589de64eec96"
```

Comment:
I would explain a bit more of the "why" here. I think we're doing this because it's the easiest way to special-case our QC validation logic at the cutover height. But just explain it so we know/remember.